src/random_test.cpp

```diff
@@ -89,13 +88,18 @@ int sum_of_all_options(Thd1 *thd) {
     ;
 
   /* for 5.7 disable some features */
-  if (db_branch().compare("5.7") == 0) {
+  if (db_branch().substr(0,3).compare("5.7") == 0) {
     opt_int_set(ALTER_TABLESPACE_RENAME, 0);
     opt_int_set(RENAME_COLUMN, 0);
     opt_int_set(UNDO_SQL, 0);
     opt_int_set(ALTER_REDO_LOGGING, 0);
   }
 
+  /* Disabling until Bug: https://jira.percona.com/browse/PS-7865 is fixed by upstream */
+  if (db_branch().substr(0,6).compare("8.0.26") == 0) {
```

Comment:
We need to disable it for lower version as well. not just only for 8.0.26.

Reply:
Fixed